site/content/en/docs/tasks/manage/enable_kubeviz.md

```diff
@@ -0,0 +1,202 @@
+---
+title: "Enable KueueViz"
+date: 2024-01-15
+weight: 3
+description: >
+  Installing and configuring KueueViz, a web-based visualization tool for Kueue workload monitoring.
+---
+
+<!-- toc -->
+
+- [Before you begin](#before-you-begin)
+- [Installation with Kueue](#installation-with-kueue)
+- [Enable KueueViz (Kueue is already installed)](#enable-kueueviz-kueue-is-already-installed)
+  - [Enable KueueViz by Helm](#enable-kueueviz-by-helm)
+  - [Enable KueueViz by YAML](#enable-kueueviz-by-yaml)
+  - [Enable KueueViz by Kustomize](#enable-kueueviz-by-kustomize)
+- [Accessing the Dashboard](#accessing-the-dashboard)
+- [Upgrade](#upgrade)
+- [Uninstall](#uninstall)
+- [What's next](#whats-next)
+
+<!-- /toc -->
```

Comment:
Do we need this? I thought we already had this navigation in the right menu.

Reply:
ok, I'll delete this part.